notebooks/adastra/data-owners/01-data-owners-start.ipynb

```diff
@@ -34,27 +36,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# edit the IP for the Network node into this variable\n",
+    "# Set the Ip Address of your Network node in this variable.\n",
+    "# The network node with IP `13.83.1.243` is managed and maintained by OpenMined.\n",
```

Comment:
I think the network ip `13.83.1.243` was not working before, should we leave it in the comments?